languages/elixir/exercises/concept/list-comprehensions/.docs/introduction.md

```diff
@@ -1,10 +1,4 @@
-Comprehensions provide a facility for transforming _Enumerables_ easily and declaratively. There are three required parts to a comprehension:
-
-- _generators_:
-  - Values enumerated from structures that implement the _Enumerable_ protocol.
-  - Pattern matching expressions to destructure enumerated values.
-- _filters_: Boolean conditions, used to select which enumerated values to use when creating the new values.
-- _collectables_: A structure which implements the _Collectable_ protocol, used to collect the new values.
```

Comment:
I remove this part because it's hard to follow without seeing any code samples first. I added an example of a filter later, and `into:` is not needed to solve this exercise, so we might skip it altogether.